presto-docs/src/main/sphinx/functions.rst

```diff
@@ -29,3 +29,4 @@ Functions and Operators
     functions/color
     functions/session
     functions/teradata
+    functions/phonenumber
```

Comment:
I'm not sure three functions warrant a separate section.

Reply:
Even I wasn't super sure if this deserves a separate function. My options when I was thinking about this were:

1) Create the phone number specific section - (this is what I did). 
2) Create some catch-all - Utility functions (?) and then add this there and we can add some of the one-off functions to this one list.
3) Don't document this function

I went on with (1) because:

1) (3) seems absurd - that would mean a non-discoverable functionality
2) my concern with (2) was loss in discoverability and also there exists none till now
3) `functions/session` is an example with just 1 function - hence I felt this is already a pattern and should be fine.

I am happy to hear your recommendation and then I can change the documentation accordingly.